x/stake/keeper/validator.go

```diff
@@ -738,3 +725,9 @@ func (k Keeper) clearCliffValidator(ctx sdk.Context) {
 	store.Delete(ValidatorPowerCliffKey)
 	store.Delete(ValidatorCliffIndexKey)
 }
+
+func ensureValidatorFound(found bool, ownerAddr []byte) {
+	if !found {
+		panic(fmt.Sprintf("validator record not found for address: %v\n", ownerAddr))
```

Comment:
Do you think we should print the hex encoding here (i.e. `%X`)? I never found the raw bytes in the logs to be useful?

Reply:
Agreed